docs/styling/THEME.md

```diff
@@ -3,42 +3,39 @@
 ## About the theme
 
 The theme consists in a set of properties (border, radius, shadow, colors) that are consistent throughout the application.
-If the airline changes one of those properties, it will impact all the elements that rely upon it. For example, if
-in the theme, a shadow color is defined, the airline can override it and impact all the components with shadows.
+If the client changes one of those properties, it will impact all the elements that rely upon it. For example, if
```

Comment:
the meaning of `client` doesn't look clear to me out of context
maybe we can find a better name or at least start with a small explanation on the term